scripts/clone_toggl_hours_to_another_account.py

```diff
@@ -12,7 +12,7 @@
 def get_entries(start, end, from_workspace_id, from_user_id):
     toggl = Toggl()
 
-    toggl.setAPIKey(os.getenv('TOGGL_APIKEY'))
+    toggl.setAPIKey(os.getenv('TOGGL_APIKEY1'))
```

Comment:
TOGGLE_APIKEY_FROM and TOGGLE_APIKEY_TO would make sense.

Reply:
Sounds maintainable indeed.